acquia_lift.admin.wizard.inc

```diff
@@ -1030,6 +1030,7 @@ function acquia_lift_personalize_campaign_wizard_submit(&$form, &$form_state) {
 
   // Run the base submit form handling.
   $agent_data = personalize_campaign_wizard_submit_base($form, $form_state);
+  $agent_data = personalize_agent_load($agent_data->machine_name);
   $agent_instance = personalize_agent_load_agent($agent_data->machine_name);
```

Comment:
You mean like this https://github.com/acquia/personalize_module/compare/CH-2470...fix-review-submit?expand=1 ?